js/src/util.js

```diff
@@ -67,6 +67,13 @@ const Util = (($) => {
 
     TRANSITION_END: 'bsTransitionEnd',
 
+    getPageDirection() {
+      const htmlElement = document.getElementsByTagName('html')[0]
```

Comment:
any reason why you don't use `document.documentElement` here?

Reply:
I was aiming for coverage and the dev docs around documentElement seemed unsure of IE and Edge support. If there's info surrounding coverage, I'd rather use it over the current implementation and would update accordingly.

Reply:
You work for Microsoft, you should be able to learn about it from the inside :P

Reply:
But joking aside, works fine for me on latest Edge that ships with 10 stable and IE 11 and IE 10 (emulated).

Reply:
Those worked for me as well. And Emulated doesn't always do the trick. #trustMe.

However, I ran it through some rough and quick tests on Browser Stack, seemed to work as expected. At least on Win10. If someone else wants to roll back to win7 and Vista and test it, please send me your findings. Making the update now.